app/models/user.rb

```diff
@@ -0,0 +1,16 @@
+class User < ApplicationRecord
+  has_many :invitations, foreign_key: :attendee_id
+  has_many :attended_events, through: :invitations
+  has_many :events
+  validates :name, :email, presence: { message: '- The field can not be empty' }
```

Comment:
The user in the events model should be referred to as the "creator" i.e. the foreign key should be `creator_id`. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/associations/user_assoc_requirement.png) from [Odin](https://www.theodinproject.com/courses/ruby-on-rails/lessons/associations))